frontend/app/templates/components/checkin/multi-select-group.hbs

```diff
@@ -0,0 +1,22 @@
+{{checkin/multi-select-edit
+  objects=relatedObjects
+  modelName=modelName
+  badgeStyle=badgeStyle
+  selectedTag=selectedObject
```

Comment:
I would suggest something more generic like `selectedItem` rather than `selectedTag` as `Tag` has a specific meaning in Flaredown